chacra/async/__init__.py

```diff
@@ -0,0 +1,117 @@
+import pecan
+from celery import Celery
+import celery
+from datetime import timedelta
+from chacra import models
+from chacra.util import repo_directory
+import os
+import logging
+logger = logging.getLogger(__name__)
+
+
+def get_pecan_config():
+    try:
+        os.environ['PECAN_CONFIG']
+    except KeyError:
+        here = os.path.abspath(os.path.dirname(__file__))
+        config_path = os.path.abspath(os.path.join(here, '../config/config.py'))
```

Comment:
this code is silly, I should've made use of the PECAN_CONFIG above but I throw that away :/ Will fix